.github/workflows/build-cli-image.yml

```diff
@@ -53,11 +56,12 @@ jobs:
 
       - name: Build and push Docker image
         id: build-and-push
-        uses: docker/build-push-action@ac9327eae2b366085ac7f6a2d02df8aa8ead720a
+        uses: depot/build-push-action@v1
         with:
           context: "."
           file: "cli/Dockerfile"
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          platforms: linux/amd64
+          platforms: linux/amd64,linux/arm64
+          buildx-fallback: true
```

Comment:
This option means that if `depot build` fails, the action will re-try the build with local `docker buildx build`